### PR TITLE
New Version: BraveSoftware.BraveBrowser.Beta 101.0.4951.41

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58698)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BraveSoftware/BraveBrowser/Beta/101.0.4951.41/BraveSoftware.BraveBrowser.Beta.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser.Beta
+PackageVersion: 101.0.4951.41
```
[the remark this answers is not in the document]
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/BraveBrowser/Beta/101.0.4951.41/BraveSoftware.BraveBrowser.Beta.installer.yaml:5]: Brave's versioning scheme is different from Chromium.

File: manifests/b/BraveSoftware/BraveBrowser/Beta/101.0.4951.41/BraveSoftware.BraveBrowser.Beta.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.5-1-19041-1645
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BraveSoftware.BraveBrowser.Beta
+PackageVersion: 101.0.4951.41
```
[the remark this answers is not in the document]
Comment (ItzLevvie) [reply manifests/b/BraveSoftware/BraveBrowser/Beta/101.0.4951.41/BraveSoftware.BraveBrowser.Beta.locale.en-US.yaml:5]: Brave's versioning scheme is different from Chromium.

**Review by OfficialEsco — APPROVED** [review]

**Comment by startadv** [comment]
I think after getting approved it should merge automatically. However it is not merging after 1 hour. Is there any issue?

**Review by ItzLevvie — APPROVED** [review]

**Comment by OfficialEsco** [comment]
I agree, but for some reason the MS team want msftbot to do the stuff, which we keep breaking, as you can see it insta merged once Levvie retriggered a event which we also don't know the reason for..

**Pull request merged**